clang/include/clang/Basic/Builtins.td

```diff
@@ -3452,9 +3452,10 @@ def Fmod : FPMathTemplate, LibBuiltin<"math.h"> {
 
 def Frexp : FPMathTemplate, LibBuiltin<"math.h"> {
```

Comment:
There is a separate `FrexpF16F128` that should probably also be updated.

Reply:
I have added `UnprefixedOnlyConstexprSince` to the class `LIBBUILTIN` so it will be unknow if I add it to this def. I will wait to see if the changes I made were correct and update this rule accordingly.